templates/contact.html

```diff
@@ -77,7 +77,7 @@ <h4 class="text-white display-5">
       </div>
       <div class="row mt-3 mr-5">
         <h4 class="text-primary">{{ _('Report a security issue.') }}</h4>
-        <p class="text-tpo">{{ _("If you've found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you've found a security bug in Tor or Tor Browser, feel free to submit it for our <a href="https://hackerone.com/torproject" target="_blank">{{ _('bug bounty program.') }}</a> If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
+        <p class="text-tpo">{{ _("If you have found a security issue in one of our projects or in our infrastructure, please email tor-security@lists.torproject.org. If you have found a security bug in Tor or Tor Browser, feel free to submit it for our <a href="https://hackerone.com/torproject" target="_blank">{{ _('bug bounty program.') }}</a> If you want to encrypt your mail, you can get the GPG public key for the list by contacting tor-security-sendkey@lists.torproject.org or from pool.sks-keyservers.net. Here is the fingerprint:") }}</p>
```

Comment:
The template call {{ _('bug bounty program.') }} is inside another {{ _(' ') }}, so it is still not building.
This pull request is a bit tricky, because it is on a template. The link will probably not get well parsed as well.
I think it is necessary to have a local setup to be able to work on templates, this way, you can build and see the errors yourself: https://dip.torproject.org/torproject/web/tpo/wikis/Compiling-a-local-version-of-the-website

Reply:
Hi @PROTechThor, I already fixed this issue. For template modifications please try before in your local lektor instance.

Reply:
Noted. Thank you!